### PR TITLE
LX-1414 Build delphix-virtualization/entire with x.x.x.x scheme

### PR DESCRIPTION
**Pull request by prakashsurya**
Co-authored-by: Prakash Surya <prakash.surya@delphix.com>

### CODE REVIEW

**Comment by prakashsurya** [comment]
`git-ab-pre-push` is [here](http://selfservice.jenkins.delphix.com/job/devops-gate/job/master/job/appliance-build-orchestrator-pre-push/122/)

File: live-build/misc/live-build-hooks/82-upgrade-repository.binary [hunk 1]
```diff
@@ -92,9 +92,32 @@ cp config/hooks/template.ctl delphix-entire.ctl
 DEPENDS=$(chroot binary \
 	dpkg-query -Wf '${Package} (=${Version}), ' | sed 's/, $//')
 
+if chroot binary dpkg -l | grep -q delphix-virtualization; then
```
Comment (dtzeng) [inline live-build/misc/live-build-hooks/82-upgrade-repository.binary]: I think we need to set pipefail for this to handle the case where the dpkg command fails?

Comment (prakashsurya) [reply live-build/misc/live-build-hooks/82-upgrade-repository.binary]: We're already doing that, right? See [here](https://github.com/delphix/appliance-build/blob/master/live-build/misc/live-build-hooks/82-upgrade-repository.binary#L27); or am I overlooking something?

Comment (dtzeng) [reply live-build/misc/live-build-hooks/82-upgrade-repository.binary]: Ah you are right, I missed that

**Pull request merged**